### DISCUSSION TITLE
A group with multiple managers and packages

### DISCUSSION THREAD

**Discussion by marksieczkowski** [renovatebot/renovate#27195] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I have a package rule like this and my goal is to have a PR with any Terraform version updates in `asdf` and the various provider updates from the `terraform` manager.  Is there a way to have the `matchPackagePatterns` only apply if it is the `asdf` manager, and no pattern match for the `terraform` manager?  Thanks   

```
{
      "matchManagers": ["asdf", "terraform"],
      "matchPackagePatterns": ["^terraform"],
      "groupName": "terraformUpdate",
      "semanticCommitType": "chore",
      "rangeStrategy": "replace"
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes, but you need to split it into two rules

**Reply by marksieczkowski** [reply]
2 Rules but same `groupName`?

**Reply by rarkins** [reply]
Yes
